Add relvals for Run3 PbPb skims

stahlleiton opened the pull request:
#### PR description:

This PR adds the following relvals for testing Run3 PbPb skims:

- 141.201: 2023 PbPb HIForward skims
- 141.202: 2023 PbPb HIPhysicsRawPrime skims
- 142.201: 2024 PbPb HIForward skims
- 142.202: 2024 PbPb HIPhysicsRawPrime skims
- 143.201: 2025 PbPb HIForward skims
- 143.202: 2025 PbPb HIPhysicsRawPrime skims

@mandrenguyen 

#### PR validation:

Tested with relvals: 141.201,141.202,142.201,142.202,143.201,143.202

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

To be backported for 2023 and 2024 PbPb skims

chrishanw committed:
Introduce new simple seed creator and ElectronSeedFitter producers for egamma pixel seeding speedup.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-49673/47226

cmsbuild commented:
A new Pull Request was created by  @stahlleiton for master.

It involves the following packages:

- Configuration/PyReleaseValidation (**pdmv**)


@AdrianoDee, @DickyChant, @antoniovagnerini, @cmsbuild, @miquork can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @fabiocos, @makortel, @slomeo this is something you requested to watch as well.
@ftenchini, @mandrenguyen, @sextonkennedy you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

cmsbuild commented:
Pull request #49673 was updated. @AdrianoDee, @DickyChant, @antoniovagnerini, @cmsbuild, @miquork can you please check and sign again.

mandrenguyen commented:
please test workflow 141.201,141.202,142.201,142.202,143.201,143.202

cmsbuild commented:
+1

**Size**: This PR adds an extra 16KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-c4b67f/50372/summary.html
**COMMIT**: e4409b9dd3ee4376fc420b216db5a7aaacdec8f6
**CMSSW**: CMSSW_16_1_X_2025-12-19-1100/el8_amd64_gcc13
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/49673/50372/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



There are some workflows for which there are errors in the baseline:
 250202.181 step 3
 The results for the comparisons for these workflows could be incomplete 
This means most likely that the IB is having errors in the relvals.The error does NOT come from this pull request

**Summary**:
 - You potentially [added 16 lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_16_1_X_2025-12-19-1100+c4b67f/72471/validateJR/logRootQA-events.log) to the logs
 - [Reco comparison results: 7 differences found in the comparisons](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_16_1_X_2025-12-19-1100+c4b67f/72471/validateJR.html)
 - Reco comparison had  4 failed jobs
 - DQMHistoTests: Total files compared: 53
 - DQMHistoTests: Total histograms compared: 4280393
 - DQMHistoTests: [Total failures: 14](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_16_1_X_2025-12-19-1100+c4b67f/72471/validateJR.html)
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 4280359
 - DQMHistoTests: Total skipped: 20
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 52 files compared)
 - Checked 227 log files, 198 edm output root files, 53 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_16_1_X_2025-12-19-1100+c4b67f/72471/triggerResults): no differences found

cmsbuild committed:
Merge pull request #49668 from cms-tsg-storm/HLTdevelopment151X_for160X

HLT menu development for `15_1_X` (4/N) [`16_0_X`]

cmsbuild committed:
Merge pull request #49672 from ghyls/devel-cmsenv-mpirun

A `cmsenv_mpirun` that works transparently with OpenMPI 5, OpenMPI 4 and MPICH, [`16_0_X`]

cmsbuild committed:
Merge pull request #49676 from SegmentLinking/initialize_t4_dBeta

LST: Ensure quadruplet 'dBeta' initialization, Backport 16_0_X

mmusich committed:
HLT Menu development for 16_0_X

- Migrate all tables to use 16_0_0_pre4 template
- update TSG scripts to use 16_0_X
- remove 15.1.X-based customizeHLTforCMSSW customization functions

mmusich committed:
update global tags to be used 2026 TSG tests

cmsbuild committed:
Merge pull request #49687 from stahlleiton/Era_2026_PbPb_CMSSW_16_1_X

[16_0_X] Add 2026 PbPb eras

cmsbuild committed:
Merge pull request #49721 from cms-tsg-storm/HLTdevelopment160X

 HLT menu development for `16_0_X` (1/N)

chrishanw committed:
Implement comments in 49436 and add HLT customisation function.

mmusich committed:
update run to be used for the scouting_dqm_sourceclient unit test

cmsbuild committed:
Merge pull request #49688 from stahlleiton/FG_CMSSW_16_0_X

[16_0_X] Update HF FG thresholds for 2025 PbPb

cmsbuild committed:
Merge pull request #49731 from chrishanw/simplified-Egamma-seed-generation

Simplified egamma seed generation - `16_0_X`

cmsbuild committed:
Merge pull request #49767 from mmusich/mm_update_DQMOnlineScouting_unitTest_16_0_X

update run to be used for the `scouting_dqm_sourceclient` unit test [16.0.X]

Commit:
Add relvals for Run3 PbPb skims

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-49673/47451
